bgpd/bgp_evpn_private.h

```diff
@@ -581,32 +581,32 @@ evpn_type2_prefix_vni_mac_copy(struct prefix_evpn *vni_p,
 static inline struct ethaddr *
 evpn_type2_path_info_get_mac(const struct bgp_path_info *local_pi)
 {
-	assert(local_pi->extra);
-	return &local_pi->extra->vni_info.mac;
+	assert(local_pi->extra->evpn);
+	return &local_pi->extra->evpn->vni_info.mac;
 }
 
 /* Get IP of path_info prefix */
 static inline struct ipaddr *
 evpn_type2_path_info_get_ip(const struct bgp_path_info *local_pi)
 {
-	assert(local_pi->extra);
-	return &local_pi->extra->vni_info.ip;
+	assert(local_pi->extra->evpn);
```

Comment:
Ditto.

Reply:
Thanks for your carefulness! It has been fixed.